### PR TITLE
[MOB-3377] consolidate message components into IterableInboxMessageCell component

### PR DESCRIPTION
**Pull request by evangreer91**
## 🔹 JIRA Ticket(s) if any

* [MOB-3377] (https://iterable.atlassian.net/browse/MOB-3377)

## ✏️ Description

This pull request consolidates the IterableInboxSwipeableRow, IterableInboxClickableRow, and IterableInboxMessageListItem into IterableInboxCell component.

[MOB-3377]: https://iterable.atlassian.net/browse/MOB-3377?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: ts/IterableInboxMessageCell.tsx [hunk 1]
```diff
@@ -139,17 +137,146 @@ const defaultMessageListLayout = (
    )
 }
 
-const IterableInboxMessageListItem = ({
+type MessageCellProps = {
+   key: string,
```
Comment (roninopf) [inline ts/IterableInboxMessageCell.tsx:141]: Can you remove unused properties as a result of this consolidation? `key`, at least, isn't used anywhere else.

Comment (evangreer91) [reply ts/IterableInboxMessageCell.tsx]: Yes, thanks.

File: ts/IterableInboxMessageCell.tsx [hunk 2]
```diff
@@ -1,25 +1,23 @@
 'use strict'
 
-import React from 'react'
+import React, { useRef } from 'react'
```
Comment (Ayyanchira) [inline ts/IterableInboxMessageCell.tsx:3]: Just a nit pick.
Not sure whats the best practice. Should we write all the modules within same package in curly braces? some outside and some inside looks unformatted.

Comment (evangreer91) [reply ts/IterableInboxMessageCell.tsx:3]: This is the convention. If I include React within the braces, I get an error that React can only be imported as a default import.

**Review by Ayyanchira — APPROVED** [review]
Tested.. Runs on device too..

**Pull request merged**